src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.WindowsSettings/Helpers/ContextMenuHelper.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Windows;
+using System.Windows.Input;
+
+using Microsoft.CmdPal.Ext.WindowsSettings.Classes;
+using Microsoft.CmdPal.Ext.WindowsSettings.Commands;
+using Microsoft.CmdPal.Ext.WindowsSettings.Properties;
+using Microsoft.CmdPal.Extensions.Helpers;
+
+namespace Microsoft.CmdPal.Ext.WindowsSettings.Helpers;
+
+/// <summary>
+/// Helper class to easier work with context menu entries
+/// </summary>
+internal static class ContextMenuHelper
+{
+    internal static List<CommandContextItem> GetContextMenu(WindowsSetting entry)
```

Comment:
weird to me that this isn't just a method on the class that builds the IListItems

Reply:
I took the same approach the OG plugin had -- I can move things around.